src/v2/ion/ui-schema/ion-boolean-handler.js

```diff
@@ -32,12 +32,13 @@ const getRadioUiSchema = ({ label, required, options }) => ({
   sublabel: required? null : loc('oie.form.field.optional', 'login'),
 });
 
-const getCheckboxUiSchemaWithDefaultValue = ({ label, type, required }) => ({
+const getCheckboxUiSchemaWithDefaultValue = ({ label, type }) => ({
   placeholder: label,
   label: false,
   modelType: type,
   type: 'checkbox',
-  required: required,
+  // set required true so default value can be passed to optional attributes as well
```

Comment:
If `required` is set to false, what happens?

Reply:
null value will be passed to the backend instead of default value. since the value is not required which means it can be empty. so after setting it as true, it will force default value as false if it is null.

Reply:
Got it, so it needs to be required so a T/F value is passed through 👌

Reply:
All good on test coverage too?

Reply:
right. 
```  await t.expect(await enrollProfilePage.getCheckboxValue('userProfile.required')).eql(false);
  await t.expect(await enrollProfilePage.getCheckboxValue('userProfile.optional')).eql(false);

Reply:
right. 
```  await t.expect(await enrollProfilePage.getCheckboxValue('userProfile.required')).eql(false);
  await t.expect(await enrollProfilePage.getCheckboxValue('userProfile.optional')).eql(false);